test/views/behaviours/IntViewBehaviours.scala

```diff
@@ -60,7 +60,7 @@ trait IntViewBehaviours extends QuestionViewBehaviours[Int] {
         "show an error summary" in {
 
           val doc = asDocument(createView(form.withError(error)))
-          assertRenderedById(doc, "error-summary-title")
+          assertRenderedByClass(doc, "error-summary-title")
```

Comment:
Change "error-summary-title" to "govuk-error-summary"

Reply:
Done